### PR TITLE
Fixes #22201: Parent breaks postinst

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/22201

### CODE REVIEW

File: rudder-agent/SOURCES/rudder-agent-postinst [hunk 1]
```diff
@@ -88,7 +88,11 @@ then
     /usr/bin/mkssys -s rudder-agent -p "${CFE_DIR}/bin/cf-execd" -a "-F" -u root -S -n15 -f9 -R
     /usr/sbin/mkitab "rudder-agent:23456789:once:/usr/bin/startsrc -s rudder-agent"
     # No need to tell init to re-read /etc/inittab, it does it automatically every 60 seconds
-    echo "0,5,10,15,20,25,30,35,40,45,50,55 /opt/rudder/bin/rudder agent check -q >> /var/log/rudder/agent-check/check.log 2>&1 # RUDDER CRON" >> /var/spool/cron/crontabs/root
+    sed -i '/# RUDDER CRON$/d' /var/spool/cron/crontabs/root > /tmp/rudder-installer-cron
```
Comment (amousset) [inline rudder-agent/SOURCES/rudder-agent-postinst:91]:
```suggestion
    sed '/# RUDDER CRON$/d' /var/spool/cron/crontabs/root > /tmp/rudder-installer-cron
```

Comment (ncharles) [inline rudder-agent/SOURCES/rudder-agent-postinst:91]: could it be in /var/rudder/tmp rather than tmp ?

Comment (peckpeck) [reply rudder-agent/SOURCES/rudder-agent-postinst:91]: -i doesn't exist on AIX

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**